package.json

```diff
@@ -30,7 +30,8 @@
     "react-datetime": "^2.16.3",
     "react-select-async-paginate": "^0.6.1",
     "react-tooltip": "^3.6.1",
-    "styled-components": "^3.2.6"
+    "styled-components": "^3.2.6",
+    "zxcvbn": "^4.4.2"
```

Comment:
afaik this library size is big. is it possible to use another library?

Reply:
we already agree to use this lib, what is the size if I may know

Reply:
799kb minified and 388kb minified + gzipped

Reply:
@yanuaracb let me know if this is unacceptable, please see the research conclusion here https://accelbyte.atlassian.net/browse/OS-7353?focusedCommentId=150602

Thank you @rayhan-ab  for raising this

Reply:
if it's not possible to use another library. could we lazy load this? so that it'll only be loaded if `hasPasswordStrenghtMeter` is true

Reply:
btw i found [zxcvbn-ts](https://www.npmjs.com/package/@zxcvbn-ts/core) and it's actively maintained. maybe you could take a look @aylwin-AB

Reply:
the size is a valid concern, please lazy load it. 
zxcvbn-ts looks promising nice find @rayhan-ab , maybe worth to compare since they state it's modernized version of zxcvbn @aylwin-AB

Reply:
nice fine @rayhan-ab, it's small too https://bundlephobia.com/package/@zxcvbn-ts/core@2.0.4, I will try that one